main.go

```diff
@@ -18,6 +20,101 @@ import (
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
+const (
+	diskPath             string = "./tmp/disk.img.gz"
+	diskPathDecompressed string = "./tmp/disk.img"
+	diskPathConverted    string = "./tmp/disk.qcow2"
+)
+
+func apply(vmName string) error {
+	log.Println("Applying VirtualMachineExport object...")
+
+	yaml := fmt.Sprintf(`
+apiVersion: export.kubevirt.io/v1alpha1
```

Comment:
Please use https://kubevirt.io/api-reference/v1.1.1/definitions.html#_v1alpha1_virtualmachineexport instead of plain string

Reply:
I'll take care of it in the another PR.